cli/handleSysCheck.js

```diff
@@ -13,13 +23,115 @@ const renderHelp = () => `
     $ yeep syscheck --config=yeep.config.js
 `;
 
+const validateMongo = async (uri, spinner) => {
+  const client = await MongoClient.connect(
+    uri,
+    { useNewUrlParser: true }
+  );
+
+  spinner.stopAndPersist({
+    symbol: chalk.green('✔'),
+    text: 'Connected to database',
+  });
+
+  const db = client.db();
+  try {
+    // TODO: Verify that the uri has the username:password format if the authentication fails!
+    const buildInfo = await db.admin().buildInfo();
+    const isVersionValid = buildInfo.versionArray[0] >= 4;
+    if (!isVersionValid) {
+      spinner.fail(`Mongo version ${buildInfo.version} needs to be higher than 4.0.0`);
```

Comment:
I suggest we move `spinner.fail()` inside the catch statement at line 76. However, we need to update the error message to exactly match the text printed to stderr. Merely a suggestion to avoid repeating spinner.fail() over and over.

Reply:
yes! thanks. Moving all the errors in the catch clause was what i was missing for removing the spinner.fails